CHANGELOG

```diff
@@ -0,0 +1,7 @@
+# ChangeLog
+## Added
+Introduces two new functions to the libmongocrypt API.
+- mongocrypt_setopt_kms_providers
+    To set the KMS providers.
+- mongocrypt_ctx_setopt_key_encryption_key
+    To set the key encryption key.
```

Comment:
Is this supposed to be a markdown file?

Reply:
Good catch, I was following https://keepachangelog.com/en/1.0.0/, which recommends naming this CHANGELOG.md.